src/Core/Manifest/ManifestFileFinder.php

```diff
@@ -32,7 +32,8 @@ class ManifestFileFinder extends FileFinder
         'include_themes' => false,
         'ignore_tests' => true,
         'min_depth' => 1,
-        'ignore_dirs' => ['node_modules']
+        'ignore_dirs' => ['node_modules'],
+        'ignore_ci_library' => []
```

Comment:
ci_library?  What's a CI library?

Reply:
"Continuous Integration library" ... I didn't want to explicitly call this `PHPUNIT_something` to be library agnostic.

I've renamed the option `ignore_ci_libraries` since it's a list

Reply:
We doesn't really use the word 'library', we usually call everything 'modules'

`ignore_test_modules` ?

Reply:
Whatever we choose here, I think it should be aligned with this other discussion https://github.com/silverstripe/silverstripe-framework/pull/10150#discussion_r750797920

If we end up renaming that other method `getTestModule`, then the option should become `ignore_test_modules`.